Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -410,15 +410,15 @@ HRESULT STDMETHODCALLTYPE GetPixelsPerDip(_In_opt_ void* clientDrawingContext, _
  * @parameter range attributed string range to use.
  * @parameter frameSize size parameters of the frame to fit the text into.
  *
- * @return _CTFrame* created using the given parameters
+ * @return CTFrameRef created using the given parameters
  */
-static _CTFrame* _DWriteGetFrame(CFAttributedStringRef string, CFRange range, CGRect frameSize) {
+static CTFrameRef _DWriteGetFrame(CFAttributedStringRef string, CFRange range, CGRect frameSize) {
     RETURN_NULL_IF(!string);
 
-    _CTFrame* frame = [[_CTFrame new] autorelease];
-    if (range.length <= 0) {
-        return frame;
-    }
+    __CTFrame* frame = __CTFrame::CreateInstance(kCFAllocatorDefault);
+    CFAutorelease(frame);
```

Comment:
I hate that this method returns an "un-owned" CTFrame. It should be a Create method or it should not exist.